.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,14 @@
+- id: terraform-docs
```

Comment:
maybe also run yamllint on this file utilizing a pre-commit hook? :-)

Reply:
Indeed that would be great! But I don't think that's really in scope for this PR, it should be something the regular maintainers decide they want to add to the workflow.